projects/kit/components/tabs/tab/tab.providers.ts

```diff
@@ -31,8 +39,15 @@ export function tabActiveFactory(
     {nativeElement}: ElementRef<HTMLElement>,
     documentRef: Document,
     routerLinkActiveService: Observable<boolean>,
+    mutationObserverService: MutationObserverService,
+    routerLinkActive: RouterLinkActive,
 ): Observable<unknown> {
+    const mutationObserver = routerLinkActive
+        ? mutationObserverService.pipe(filter(() => routerLinkActive.isActive))
```

Comment:
don't we need to throttle it to prevent an endless loop in some unpredictable cases? 🤔